BCStudentSoftwareDevTeam/celts
# Views count

fixed issue #856, add a view counter on the event view page to keep track of the people who view an event. 

## Testing
-  checkout branch viewsCount
-  reset the database 
-  make sure tests are not failing 
-  try to view an event (not admin)
-  switch the current user to admin and see if the view is counted 
-  only the admin can see the views, admin views do not count, and no duplicate views 
-  check if the EventView table is updated
